packages/core/core/test/RequestTracker.test.js

```diff
@@ -16,13 +16,15 @@ describe('RequestTracker', () => {
     let tracker = new RequestTracker({farm, options});
     await tracker.runRequest({
       id: 'abc',
+      // $FlowFixMe[incompatible-call]
```

Comment:
nit: It might be better to build a mock typecast so there's no any type created by the suppression, which would mean the test won't break if the types change

Something like:
```
async function mockRunRequest(
      tracker: RequestTracker,
      request: {...Request, type: Request.type | 'mock_request', ...},
      opts?: ?RunRequestOpts,
    ): Promise<TResult> {
      return tracker.runRequest(request, opts);
    }
```

Reply:
Good call @JakeLane! Let me know what you think about how I mocked `runRequest` (wasn't sure if that was the best way)

Reply:
@JakeLane just FYI I ended up changing all the `mock_request`s to be one of the types in the `RequestType` union. When I tried mocking the types (https://github.com/parcel-bundler/parcel/pull/9383/commits/53fdf808757413c9fb5d3dbf2de2e4f0f7ab91a5), the only way I found that appeased Flow was to do it in a way that essentially did the same thing (as above) 😅